cli/src/commands/fix.rs

```diff
@@ -256,6 +272,15 @@ fn run_tool(
     })
     .unwrap()?;
     tracing::debug!(?command, ?output.status, "fix tool exited:");
+    if !output.stderr.is_empty() {
+        let prefix = format!(
+            "{}:\n",
+            path_converter.format_file_path(&file_to_fix.repo_path)
+        );
+        let mut stderr = ui.stderr();
+        stderr.write_all(prefix.as_bytes()).ok();
+        stderr.write_all(&output.stderr).ok();
+    }
```

Comment:
maybe better to do `writeln!(stderr)` if stderr isn't terminated by newline?

Reply:
I can use `writeln!` for the file name, but it's not guaranteed stderr is UTF-8, and I think using `writeln!` needs the format args to be Display or Debug, unless I'm missing something.

What do you think about just emitting a `\n` after we write to stderr? Like this:

```rust
writeln!(stderr, "{}:", path_converter.format_file_path(&file_to_fix.repo_path)).ok();
stderr.write_all(&output.stderr).ok();
stderr.write_all(b"\n").ok(); // or writeln!(stderr, ""), if that's more idiomatic, but it looks odd to me
```

Reply:
`format_file_path()` returns a `String`, so it can be passed to `writeln!(..)`.

> stderr.write_all(b"\n").ok(); // or writeln!(stderr, ""), if that's more idiomatic, but it looks odd to me

`writeln!(stderr)` is often used.